fares-data-build-tool/create-data
# fix: max group size not stored

- Now storing group size and updating group size attribute when retrieving stored groups
- Added a make command to install all node dependencies
- Moved the PR template to the root of the repo (I think this will kick in for the next PR?)

**To Test**
- Create a new group, reuse that group and then change the details of the group using the confirmation page change links - this should now work as expected.
